### DISCUSSION TITLE
Waters RAW files

### DISCUSSION THREAD

**Discussion by sandralu19** [mzmine/mzmine#1502] [Q&A]
Hi!

I am new in the world of metabolomics and I think I need a little bit of help with the basics. I have .RAW Waters files with an approximate size of 1.6 GB. After converting the files to MZML using MSConvert, my new .mzml files are still around the same size, which makes it difficult to work with the data even with 32GB of RAM. 

I have been trying to look for other ways to convert, including sub setting the data in MSConvert, but I get some extra peaks detected that aren't in the original file, so I'm not sure that is a good choice for me. 

Does anyone have any suggestions on how to handle .RAW Waters files?

Thanks,
Sandra

**Comment by greencodefairy** [comment]
Hi Sandra, have you tried this script? https://github.com/AnP311/Waters2mzML
MassLynx raw data is known to cause issues, because of it's file structure. I had the same problem and used the script mentioned above.
Original adata files are around 400 MB, mzML around 200 KB.
